components/customs/multi-select.tsx

```diff
@@ -128,11 +129,11 @@ export function MultiSelect(props: MultiSelectProps) {
                         const newValue = selectables.find(
                           // CommandItem appears to strip starting and ending whitespace from the value given as a key
                           // so the trim() fixes a situation where the value has some starting or trailing whitespace
-                          // and isn't recognized as a selectable option as a result.
-                          (option) => option.value.trim() === value,
+                          // and isn't recognized as a selectable option as a result. This is the same for the toLowerCase().
+                          (option) => option.label.trim().toLowerCase() === value.trim().toLowerCase(),
```

Comment:
Kind of interesting but this component appears to be using the label provided, not the value so I'm pulling the label value that the component is giving me and trying to find the value associated.